services/apis/fence/fenceProps.js

```diff
@@ -422,6 +431,9 @@ module.exports = {
     client: new Client({
       envVarsName: 'CLIENT',
     }),
+    clientb: new Client({
```

Comment:
why do you need a new basic client?
and i realize this is nitpicky but could you rename `clientImplicit.envVarsName` to not be `CLIENTB` for clarity?

Reply:
For the tests, I want to configure the clients to have different access and be able to test access/denied. You can checkout the comments in centralizedAuth.js and also the common-users PR with the arborist block for more info on the diff permissions. I could rename to `anotherBasicClient` or something, or `basicClient2` if you think that'd improve clarity

Reply:
oh sorry, i didn't notice the clients were defined in the user.yaml so i thought they had the same access
for the renaming, i meant this line https://github.com/uc-cdis/gen3-qa/blob/48b92f4ca0b2806332a099f2ac395fe2fd017f98/services/apis/fence/fenceProps.js#L442

Reply:
you could also add comments in here about which client has which policies in the user.yaml, similar to what we did for users https://github.com/uc-cdis/gen3-qa/blob/master/utils/user.js#L105

Reply:
oh, yes, good idea! and I just relooked at this, I did mess up and put clientb for ABC_CLIENT. I will fix